easybuild/easyconfigs/s/SeqLib/SeqLib-1.1.2-GCC-8.2.0-2.31.1.eb

```diff
@@ -0,0 +1,71 @@
+easyblock = 'ConfigureMake'
+
+name = 'SeqLib'
+version = '1.1.2'
+
+homepage = 'https://github.com/walaj/SeqLib'
+description = """ C++ htslib/bwa-mem/fermi interface for interrogating sequence data """
+
+toolchain = {'name': 'GCC', 'version': '8.2.0-2.31.1'}
+toolchainopts = {'cstd': 'c++11', 'pic': True}
+
+extract_cmd_pattern = 'tar -C %s/%s --strip-components=1 -xf %%s'
+seqlibdir = '%(name)s-%(version)s'
+
+source_urls = ['https://github.com/walaj/SeqLib/archive/']
+sources = [
+    'v%(version)s.tar.gz',
+    {
+        'source_urls': ['https://github.com/walaj/fermi-lite/archive/'],
+        'download_filename': '5bc90f8d70e2b66184eccbd223a3be714c914365.tar.gz',
+        'filename': 'fermi-lite-20160901.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (seqlibdir, 'fermi-lite'),
+    },
+    {
+        'source_urls': ['https://github.com/walaj/bwa/archive/'],
+        'download_filename': 'c02766e3c34ac3f4af9842e20a54b7f9f4b36d0b.tar.gz',
+        'filename': 'bwa-20161013.tar.gz',
+        'extract_cmd': extract_cmd_pattern % (seqlibdir, 'bwa'),
+    },
+    {
+        'source_urls': ['https://github.com/samtools/htslib/archive/'],
+        'download_filename': '49fdfbda20acbd73303df3c7fef84f2d972c5f8d.tar.gz',
+        'filename': 'htslib-20170620.tar.gz',
```

Comment:
@smoors Can't we include `HTSlib` as a proper dependency instead?

Reply:
@boegel we could, but I doubt it's a good idea. The version they use by default is very old (2017), and I don't want to risk breaking changes with the new version.